CoffeeChat/Controller/TalkingPointsViewController.swift

```diff
@@ -97,10 +88,79 @@ class InterestsGroupsViewController: UIViewController {
         skipButton.addTarget(self, action: #selector(skipButtonPressed), for: .touchUpInside)
         view.addSubview(skipButton)
 
-        displayedInterestsGroups = interestsGroups
-
         setupConstraints()
     }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        getAllGroups()
```

Comment:
could we potentially create a `getAllTalkingPoints()` function that calls `getAllGroups` and `getAllInterests`? at first glance i was kinda confused what was going on with groups and interests being in this file.